default_xen_pvblock_env.txt

```diff
@@ -0,0 +1,11 @@
+android_keymint_needed=N
+blk_deivce_if=pvblock
+blk_device_id=0
+bootcmd=run update_bootargs; run bootcmd_android
+bootcmd_android=verified_boot_android ${blk_deivce_if} ${blk_device_id}#misc
+bootdelay=3
+fdtaddr=0
+stderr=hypervisor
+stdin=hypervisor
+stdout=hypervisor
+update_bootargs=fdt addr ${fdtcontroladdr}; fdt get value bootargs /chosen bootargs;
```

Comment:
I'm curious - why do you need this? I believe that kernel can read bootargs from the device tree by itself.